IDR/notebook/scripts/nbconverted/1.describe_screens.py

```diff
@@ -140,15 +165,65 @@ def describe_screen(screen_id):
                 # Clean channels value and separate into stain and stain target
                 stain = set()
                 stain_target = set()
-                for entry in channels.split(";"):
-                    temp_list = entry.split(":")
-                    stain.add(temp_list[0])
-                    stain_target.add(temp_list[1])
+                for channel in channels.split(";"):
+                    # For channel entries with 'stain:target' format
+                    if bool(re.search(r"([\:])+", channel)):
+                        # Separate stain and target
+                        st_list = channel.split(":")
+                        # Add to respective sets
+                        stain.add(st_list[0])
+                        stain_target.add(st_list[1])
+
+                    # For channel entries with 'stain (target)' format
+                    elif bool(channel[channel.find("(") + 1 : channel.rfind(")")]):
+                        # Search for target name within parentheses
+                        target_name = channel[
+                            channel.find("(") + 1 : channel.rfind(")")
+                        ]
+                        # Remove parentheses for stain name
+                        stain_name = re.sub(r"\([^()]*\)", "", channel)
+                        # Remove redundancies (ie. if (target (abbreviation)) present)
+                        processed = {"target": target_name, "stain": stain_name}
+                        for key in processed.keys():
```

Comment:
Same comment about more descriptive variable names than `key` and `value` - this will help a bunch in the future.

Reply:
The variable names `key` and `value` were changed to `attribute` and `att_name`, respectively.

Added in 5496254ddce2e5b9ac546300eb4c67471deac700.

Reply:
how about `image_attribute` and `image_att_name`?

Reply:
I changed them according to your suggestion.

Added in 7b796093da48fc4446fcff47a3ec4ba62b2aab50.